### DISCUSSION TITLE
How to identify correct id number for transformation command?

### DISCUSSION THREAD

**Discussion by Anmol-Bakshi** [neperfepx/neper#869] [Q&A] [answered]
this command ran:
 neper -T -n from_morpho -morpho "size:file(size.txt),centroid:file(coordinates.txt)" -dim 2 -ori "file(orientations.txt,des=euler-bunge)" -oridescriptor "euler-bunge:active" -domain "square(8.56,1.55)" -transform "cut(ecylinder(4.76,1.5,0,0,0,1,1,0,0,0,1,0,0.5,0.25))" -o simulation-with-notch -id 30

But when I changed the -id to 2/3 it did not ran, i.e.,:
neper -T -n from_morpho -morpho "size:file(size.txt),centroid:file(coordinates.txt)" -dim 2 -ori "file(orientations.txt,des=euler-bunge)" -oridescriptor "euler-bunge:active" -domain "square(8.56,1.55)" -transform "cut(ecylinder(4.76,1.5,0,0,0,1,1,0,0,0,1,0,0.5,0.25))" -o simulation-with-notch -id 2
The above didn't ran and showed error "Error  :     > Projection failed.  Increase -n or try other -ids."

So, how to identify the correct id number?

**Comment by rquey** [comment, marked as answer]
`-transform cut` is "experimental", which means that it may work for some `-id` values and fail for others.

**Reply by Anmol-Bakshi** [reply]
Thanks a lot for replying. 
But that means, it is purely a hit and trial method to identify the -id values for which -transform cut might work ?

**Reply by rquey** [reply]
Yes, it is.
